ANSWERS.md

```diff
@@ -4,3 +4,18 @@
 3. What is an event listener?
 4. Why would we convert a NodeList into an Array?
 5. What is a component? 
+
+1. DOM stands for Document Object Model, it is an application programming interface (API) for valid HTML and well-formed XML documents. 
+It defines the logical structure of documents and the way a document is accessed and manipulated.
+
+2. An event is how js interacts with html. They can be user generated, like mouse clicks or generated by programs such as api's. 
+It is basically a change in the DOM caused by a user or an api.
+
+3. An event listener is a method used to capture events. The application is setup to respond to these events.
+
+4. NodeList is only array like. Which means it doesnt have access to all the methods that an array does.
+
+5. Component is a modular piece of software. It is supposed to be able to stand on its own. It is a piece of software that can
+be reused in other applications with no modification  or very little modifications. For example, i wrote an application for a 
+finance company. One of my coworkers requested that i make it modular because he wanted to be able to take the software and sell
```

Comment:
Great answers William. I like how in depth you went, even adding a personal experience with one of the questions. Your answers demonstrate technical familiarity.